go/apps/api/routes/v2_keys_add_permissions/handler.go

```diff
@@ -56,7 +58,9 @@ func (h *Handler) Handle(ctx context.Context, s *zen.Session) error {
 		return err
 	}
 
-	key, err := db.Query.FindLiveKeyByID(ctx, h.DB.RO(), req.KeyId)
+	key, _, err := h.LiveKeyCache.SWR(ctx, req.KeyId, func(ctx context.Context) (db.FindLiveKeyByIDRow, error) {
+		return db.Query.FindLiveKeyByID(ctx, h.DB.RO(), req.KeyId)
+	}, caches.DefaultFindFirstOp)
```

Comment:
you have to look at the `_` which is the hit parameter. 

Hit can be NULL which means a key doesn't exist